scripts/makei

```diff
@@ -164,10 +167,12 @@ def add_cvtsrcpf_parser(subparsers: argparse.ArgumentParser):
 
     cvtsrcpf_parser.add_argument(
         "-c",
-        help='the EBCDIC CCSID that the source in this directory should be compiled with. CCSID used to compile the source in this directory. If not specified, then the CCSID of the JOB running the build will be used.',
-        metavar='<CCSID>',
-        type=int,
+        "--ccsid",
+        help='The TGTCCSID for the build. If not specified, then the CCSID of the SRC-PF will be used.',
```

Comment:
The old description was better
The EBCDIC CCSID that the source in this directory should be compiled with.   If not specified, then the CCSID of the SRC-PF being converted will be used.  If that CCSID is 65535 or an invalid CCSID is encountered than the CCSID of the JOB running the build will be used.'